finagle-mysql/src/main/scala/com/twitter/finagle/mysql/Request.scala

```diff
@@ -258,6 +231,24 @@ case class ExecuteRequest(
     }
 }
 
+object ExecuteRequest {
+  def apply(stmtId: Int,
```

Comment:
style nit, drop params to new line:

    def apply(
       stmtId: Int,
       ...
    ): ExecuteRequest = ...